plugin.video.arteplussept/addon.xml

```diff
@@ -37,21 +36,21 @@ https://github.com/known-as-bmf/plugin.video.arteplussept/issues
 Contributions are welcome:
 https://github.com/known-as-bmf/plugin.video.arteplussept
         </description>
-        <description lang="it">
+        <description lang="it_IT">
 Guarda video da Arte +7
 Per richiedere nuove funzionalità o segnalare problemi:
 https://github.com/known-as-bmf/plugin.video.arteplussept/issues
 Nuovi contributi sono ben accetti:
 https://github.com/known-as-bmf/plugin.video.arteplussept
         </description>
-        <description lang="pl">
+        <description lang="pl_PL">
 Oglądaj wideos z Arte +7
 W przypadku problemów lub próśb o ulepszenie:
 https://github.com/known-as-bmf/plugin.video.arteplussept/issues
 Każda pomoc mile widziana:
 https://github.com/known-as-bmf/plugin.video.arteplussept
         </description>
-        <language>fr de en it pl</language>
+        <language>fr_FR de_DE en_GB it_IT pl_PL</language>
```

Comment:
please keep the old values, they don't really apply here (only for translation/language addons)

Reply:
Thank you for this advice. I changed back to 2-digit language codes.
I also introduced a few fixes I missed in v1.1.8 :
  - renaming of synchProgress to synch_progress in player.py
  - setting info and warning log level for player logs
  - adding details to CONTRIBUTING